### PR TITLE
0.10 refactor vite

### PR DESCRIPTION
**Pull request by Severino**
Replaced laravel mix with vite as a build tool.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -185,10 +185,10 @@ If you want to send mails to your users, you have to adjust the `MAIL_*` setting
 ### Configure JavaScript
 Spacialist is based on several JavaScript libraries, which are bundled using Webpack (configuration is done using Laravel Mix, a webpack-wrapper for Laravel). Only the zipped releases contain the already bundled JavaScript libraries. All other users have to run webpack to bundle these libraries.
```
Comment (Severino) [inline INSTALL.md:186]: This section is outdated now.

File: package.json [hunk 2]
```diff
@@ -38,10 +37,12 @@
         "eslint": "^8.51.0",
         "eslint-plugin-vue": "^9.17.0",
         "laravel-mix": "^6.0.49",
```
Comment (Severino) [inline package.json:39]: laravel-mix should be removed

File: package.json [hunk 3]
```diff
@@ -14,11 +14,9 @@
     },
     "homepage": "https://github.com/DH-Center-Tuebingen/Spacialist#readme",
     "scripts": {
-        "dev": "npm run development",
-        "development": "npx mix",
-        "watch": "npx mix watch",
-        "prod": "npm run production",
-        "production": "npx mix -p",
+        "dev": "vite",
```
Comment (v1r0x) [inline package.json:17]: We should also add a dev build for `IS_OPEN=true`

File: vite.config.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+// import react from '@vitejs/plugin-react';
+import vue from '@vitejs/plugin-vue';
+import { fileURLToPath, URL } from 'url';
+
+const isOpen = process.env.IS_OPEN === 'true';
+
+export default defineConfig({
+    plugins: [
+        laravel([
+            (isOpen) ? 'resources/js/app.js': 'resources/js/open.js',
```
Comment (v1r0x) [inline vite.config.js:12]: This has to be either `(!isOpen)` or flipped, right?

**Comment by v1r0x** [comment]
I fixed what already mentioned in my comments as well as some other stuff to get it working on instances behind a proxy.

In some cases/machines there is a problem with `dayjs`. The plugins should be available on `dayjs` object (e.g. `dayjs.utc()`), but that's not working in some cases. Then, `dayjs.default.utc()` is required. I don't know if this is related due to switching from `require()` to `import`?
It's working on my local machine with npm 8.3.1, but not on a server with npm 8.18.0 :man_shrugging:

**Pull request merged**